### PR TITLE
Fix container create

### PR DESCRIPTION
**Pull request by ngmiller**
### What

Fixes container creates client-side. We first make sure that the Content-Type header is `application/json` in the dockerService $http call. Then, we let the $http service handle the JSON encoding.

### Dependency

This will fix container creates across the board with https://github.com/lighthouse/lighthouse/pull/49

### CODE REVIEW

**Comment by ngmiller** [comment]
@theMagicalKarp @zach-taylor

**Comment by robertsheehy-wf** [comment]
+1

File: app/js/docker/dockerService.js [hunk 1]
```diff
@@ -91,7 +94,7 @@ function dockerService($http, actions, flux, alertService, configService) {
     return {
         'containers': {
             'inspect': d('GET', '/containers/{id}/json', actions.inspectContainer),
-            'create': d('POST', '/containers/create', actions.createContainer),
+            'create':  d('POST', '/containers/create', actions.createContainer),
```
Comment (robertsheehy-wf) [inline app/js/docker/dockerService.js]: One of those people aye?

Comment (ngmiller) [inline app/js/docker/dockerService.js]: Yes -_-

**Pull request merged**